### ISSUE TITLE
Improve Developer and User documentation

### ISSUE THREAD

**Issue by sthaha** [sustainable-computing-io/kepler-operator#153]
Please feel free to break these into sub-tasks

- [ ] Document how to upgrade to  0.6.1 version of operator
- [ ] Document how to make a release 
- [ ] Document how to get started with local development
- [ ] Document how SSA works

**Comment by Srishtihere** [comment]
hey I would like to work on the task no. 2. Can you assign it to me

**Comment by sthaha** [comment]
@Srishtihere , thanks a lot 🤗 . I have created https://github.com/sustainable-computing-io/kepler-operator/issues/189

**Comment by MaryAyobami** [comment]
Hello @sthaha  I'd love to work on the 3rd task. Can I proceed with that?

**Comment by sthaha** [comment]
@MaryAyobami absolutely, thanks for the help. Please feel free to reuse - https://github.com/rhobs/observability-operator/blob/main/docs/user-guides/server-side-apply.md in case that helps. 
I have created #196

**Comment by MaryAyobami** [comment]
Alright

**Comment by neelshah2409** [comment]
Hey @sthaha I would like to work on the 1st task !

**Comment by sthaha** [comment]
@neelshah2409 Thanks for helping out, but I feel that task that should be done by someone who has made the release to avoid unnecessary toil :)

**Comment by vprashar2929** [comment]
closing as not relevant anymore
